src/mlte/measurement/external_measurement.py

```diff
@@ -4,32 +4,59 @@
 
 from __future__ import annotations
 
-from typing import Type
+from typing import Type, Callable, Optional
+import typing
 
 from mlte.value import Value
 from .measurement import Measurement
 
 
 class ExternalMeasurement(Measurement):
-    def __init__(self, identifier: str, value_type: type):
+    @typing.no_type_check
+    def __init__(
+        self,
+        identifier: str,
+        value_type: type,
+        function: Optional[Callable] = None,
```

Comment:
Question: we can't annotate the type for this callable any further, correct? The signature of the callable is totally opaque to us?

Reply:
Yeah, that is the point. Just a function we call that can be anything they want. I don't see how to further restrict it since what we want is to give them freedom there.